css-view-transitions-2/Overview.bs

```diff
@@ -598,20 +615,19 @@ When capturing the old or new state for an element, perform the following steps
 			Note: this means that calling {{Document/startViewTransition()}} while capturing the old document for a cross-document view-transition would run the callback but skip the animation.
 
 		1. Let |outboundTransition| be a new {{ViewTransition}} object in |oldDocument|'s [=relevant Realm=],
-			whose [=ViewTransition/active types=] is |resolvedRule|, and whose [=ViewTransition/process old state captured=] is set to the following steps:
+			whose [=ViewTransition/active types=] is |resolvedRule|.
 
-			Issue: should we check for the opt-in again, in case there was a CSSOM change in a requestAnimationFrame callback?
+		1. Run the following steps when [=Perform pending transition operations|performing pending transition operations=]:
```

Comment:
Since we're dealing with task sources in this spec, good to document in the "setup-cross-doc-view-transition-on-old-doc" algo that it runs on the rendering task source.

Reply:
It's not though... it's running on the navigation and traversal task source.
Ack regarding calling the callback when skipped.